apis/datasciencecluster/v1/datasciencecluster_types.go

```diff
@@ -60,11 +60,11 @@ type Components struct {
 	Kserve kserve.Kserve `json:"kserve,omitempty"`
 
 	// CodeFlare component configuration.
-	// Require CodeFlare Operator to be installed before enable component
+	// If CodeFlare Operator is not already installed, it'll be installed on the fly.
```

Comment:
I think it should be:
If CodeFlare Operator has been installed in the cluster, it should be uninstalled first before enabled component.

Reply:
done